swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -127,103 +187,77 @@ service /covid/status on new http:Listener(9000) {
 In this code:
 
 - Unlike normal functions, resource methods can have accessors. In this case, the accessor is set to `get`, which means only HTTP `GET` requests could hit this resource. Ballerina automatically serializes Ballerina records as JSON and sends them over the wire. 
-- The default HTTP response status code for a resource method other than `post` is `200 OK`. For an HTTP `POST` resource, the default HTTP response status code is `201 Created`. 
+- The default HTTP response status code for a resource method other than `post` is `200 OK`. The default HTTP response status code for an HTTP `POST` resource is `201 Created`. 
 
 ### Create the second resource to add data
 
-To create the second resource of the first endpoint to add new COVID-19 data to the dataset by ISO code, add the code below to the API template file (i.e., `service.bal`).
+Create the second resource of the first endpoint to add new data to the dataset by ISO code, using the Ballerina HTTP API Designer in VS Code, as shown below.
+
+> **Note:** Define an HTTP resource, which allows the `POST` operation on the resource path `countries` and accepts a `CovidEntry[]`-typed parameter named `covidEntries`, corresponding to the payload. Upon successful creation of the entries, the response shoud be a `201` response with payload of structure `CovidEntry[]`. This can be incorporated in the return type by selecting `CovidEntry[]` as the default (`201 Created`) response type. Additionally, in case of conflicting entries, the resource should return a `409 Conflict` response with a `string` payload. This can be included in the return type by generating a `409` [status code response](/learn/by-example/http-send-different-status-codes-with-payload/) by selecting the response status code as `409 - Conflict`, the payload type as `string`, and providing a name (`IsoCodeConflict` ) for the status code record.
```

Comment:
```suggestion
> **Note:** Define an HTTP resource, which allows the `POST` operation on the resource path `countries` and accepts a `CovidEntry[]`-typed parameter named `covidEntries`, corresponding to the payload. Upon successful creation of the entries, the response shoud be a `201` response with payload of structure `CovidEntry[]`. This can be incorporated in the return type by selecting `CovidEntry[]` as the default (`201 Created`) response type. Additionally, in case of conflicting entries, the resource should return a `409 Conflict` response with a `string` payload. This can be included in the return type by generating a `409` [status code response](/learn/by-example/http-send-different-status-codes-with-payload/) by selecting `409 - Conflict` as the response status code and `string` as the payload type and providing a name (`IsoCodeConflict` ) for the status code record.
```